### PR TITLE
add "shifuctl install" to shifuctl

### PR DESCRIPTION
**Pull request by saiyan86**
**What this PR does / why we need it**:

This PR adds ```shifuctl install``` function. It checks for shifu's dependency, and if the dependencies are met,

**Will this PR make the community happier**? 

Yes. This should help new developers to onboard shifu easier.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**How is this PR tested**
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify): manual test

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/shifuctl/cmd/install/install.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+	// If there's no error, then prompt is empty
+	if prompt == "" {
+		prompt = "\033[1;32mPrerequisite met, automatically installing Shifu!\033[0m\n"
+	}
+	fmt.Println(prompt)
+
+	s.Stop()
+
+	if err := installShifu(); err != nil {
+		fmt.Println("\033[1;31mError installing Shifu\033[0m, Error: ", err)
```
Comment (kris21he) [inline pkg/shifuctl/cmd/install/install.go:46]: use zlog?

**Review by kris21he — COMMENTED** [review]
shall we add README.md to illustrate what this is and how to use it?

File: pkg/shifuctl/cmd/install/install.go (hunk 1 above)
Comment (saiyan86) [reply pkg/shifuctl/cmd/install/install.go:46]: This is a command line utility, fmt should be sufficient?

**Review by BtXin — DISMISSED** [review]
Looks good, fix the conflict then we can merge.

**Review by tomqin93 — APPROVED** [review]

**Pull request merged**